components/calendar/index.tsx

```diff
@@ -23,13 +23,12 @@ function zerofixed(v: number) {
 }
 
 export type CalendarMode = 'month' | 'year';
-
 export interface CalendarProps {
   prefixCls?: string;
   className?: string;
   value?: moment.Moment;
   defaultValue?: moment.Moment;
-  mode?: CalendarMode;
+  mode: CalendarMode;
```

Comment:
`mode` still need to be optional.

Reply:
I remove this optionality because your previews comment about type it will be required i return this change and added simple change for type. Please take a look.